PocketKit/Sources/Analytics/Context/UIContext.swift

```diff
@@ -102,6 +102,12 @@ extension UIContext {
         case tagsSaveChanges = "tagsSaveChanges"
         case externalApp = "external_app"
         case saveExtension = "save_extension"
+        case sortFilterSheet = "sort_filter"
+        case sortByNewest = "sortByNewest"
+        case sortByOldest = "sortByOldest"
+        case sortByLongest = "sortByLongest"
+        case sortByShortest = "sortByShortest"
+        case navigationDrawer = "drawer"
```

Comment:
This identifier will be discussed with Analytics for a final decision.

Reply:
discussed with Sreejita that these identifiers looks good